src/web/src/asset_DELETE.ecpp

```diff
@@ -57,11 +57,11 @@ bool database_ready;
     {
         std::string id = request.getArg("id");
         if ( id.empty() ) {
-            http_die("request-param-required", "id");
+            http_die("request-param-required", TRANSLATE_ME ("id"));
```

Comment:
are we going to translate ids?

Reply:
Definitely not, otherwise whole system would need to run translations as some parts of the code are using IDs to determine type, to get DB index, etc.